pages/clustering/high-availability.mdx

```diff
@@ -438,6 +438,12 @@ With this unit-based tracking system, the replica is expected to report progress
 every 2–3 seconds. Given this, a timeout of 60 seconds is set to avoid
 unnecessary network instability while ensuring responsiveness.
 
+Except timeouts on read and write operations, Memgraph also has a timeout of 5s
+for sockets when establishing connection. Such a timeout helps in having low p99
+latencies when using RPC stack which manifests for users as smooth and predictable
+network communication between instances.
```

Comment:
```suggestion
Except for timeouts on read and write operations, Memgraph also has a timeout of 5s
for sockets when establishing a connection. Such a timeout helps in having a low p99
latencies when using the RPC stack, which manifests for users as smooth and predictable
network communication between instances.
```

Reply:
feel free to merge the suggestion. The PR should be merged into main, not ha-timeouts